Source/WebCore/platform/graphics/ComplexTextController.cpp

```diff
@@ -58,18 +58,18 @@ class TextLayout {
     }
 
     TextLayout(RenderText& text, const FontCascade& font, float xPos)
-        : m_font(font)
+        : m_fontCascade(font)
```

Comment:
I'd also rename the constructor parameter. Ditto below.

Reply:
Oops, I did but apparently I haven't saved it. Thanks!

Reply:
You can probably WTFMove() btw

Reply:
`FontCascade` is not movable at the moment so it may not be worth the effort to revisit call sites to pass an r-value reference.